scripts/qa_crowbarsetup.sh

```diff
@@ -4289,7 +4289,7 @@ function oncontroller_testpreupgrade
 function oncontroller_testpostupgrade
 {
     # retrieve the ping results
-    local fips=$(openstack floating ip list -f value -c "Floating IP Address")
+    local fips=$(openstack ip floating list -f value -c IP)
```

Comment:
thanks for pointing out @s-t-e-v-e-n-k . I found this problem during the post upgrade step so I'm wondering if the controller did not got upgraded so it failed here. I'm trying to reproduce it to investigate further.